readme.txt

```diff
@@ -335,7 +335,8 @@ Please see the changelog for the complete list of changes in this release. Remem
 * Fix - Fixed Post ID not being sent to the_title filter for Organizers and Venues (props Anna L.) [85206]
 * Fix - Fixed issue where Month View tooltips would often go off-screen in some smaller viewport sizes [65136]
 * Fix - Fixed an issue that would sometimes render Event Aggregator options invalid even with a valid license [78469]
-* Fix - Fixed an issue where the mobile.php template file would often fail to include an event's featured image [74291]
+* Fix — Fixed an issue where the mobile.php template file would often fail to include an event's featured image [74291]
```

Comment:
Not sure why this was flipped to fancy hyphens ... be nice to keep everything consistent though :smile: